oe_theme.theme

```diff
@@ -321,6 +321,15 @@ function oe_theme_preprocess_input__date(array &$variables): void {
   $variables['placeholder'] = $ecl_format;
 }
 
+/**
+ * Implements template_preprocess_input__submit().
+ */
+function oe_theme_preprocess_input__submit(array &$variables): void {
+  if (isset($variables['element']['#variant'])) {
```

Comment:
Looking at the `Drupal\Core\Render\Element\Button` and it seems the element already supports a #button_type property. So we can use that instead and map it to the ECL "variant"